src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1157,7 +1157,8 @@ static bool processContextAttributeVector (ContextElement*
                                            double&                                    coordLat,
                                            double&                                    coordLong,
                                            std::string                                tenant,
-                                           const std::vector<std::string>&            servicePathV)
+                                           const std::vector<std::string>&            servicePathV,
+                                           std::string*                               why)
```

Comment:
What we gain in is very close to zero, nothing to worry about.
It should not be passed as reference, as it is an 'output variable' (as we saw not so long ago).
I can change it to an integer, sure, but it will be a lot harder to read the code (especially when you have to go to another file to see the definition).